src/app/commands/cmd_cut.cpp

```diff
@@ -30,12 +37,18 @@ CutCommand::CutCommand()
 
 bool CutCommand::onEnabled(Context* ctx)
 {
-  return App::instance()->inputChain().canCut(ctx);
+  return App::instance()->isGui() ?
```

Comment:
Use `ctx->isUIAvailable()` when possible:
```suggestion
  return ctx->isUIAvailable() ?
```